addons/website/models/website.py

```diff
@@ -1661,7 +1661,7 @@ def _trigram_enumerate_words(self, search_details, search, limit):
         :param limit: maximum number of records fetched per model to build the word list
         :return: yields words
         """
-        match_pattern = r'[\w-]{%s,}' % min(4, len(search) - 3)
+        match_pattern = r'[\w./-]{%s,}' % min(4, len(search) - 3)
```

Comment:
The same pattern update should be applied within `_basic_enumerate_words` too for consistency.

Reply:
@bso-odoo The updates are also applied on _basic_enumerate_words  ;) 
Thank you !